lib/inferno/dsl/fhir_resource_validation.rb

```diff
@@ -300,7 +300,8 @@ class CliContext
         CLICONTEXT_DEFAULTS = {
           sv: '4.0.1',
           doNative: false,
-          extensions: ['any']
+          extensions: ['any'],
+          disableDefaultResourceFetcher: true
```

Comment:
How confident are you that this will end up being the name for this option?

Reply:
Good point - I'm fairly confident but not 100% sure on that. The name has already changed once. With that in mind, maybe we should leave this PR open until the other PR is officially merged

Reply:
Update: the other PR has been merged: https://github.com/hapifhir/org.hl7.fhir.core/pull/1526